scss/civicrm/search-builder/_search-builder.scss

```diff
@@ -0,0 +1,44 @@
+.page-civicrm-contact-search-builder .crm-accordion-body {
+  #map-field td {
+    padding: 10px 20px 4px !important;
+  }
+
+  h3 {
```

Comment:
isn't this element styled here `civicrm/dashboard`? can we make it common to make is useful on multiple places?